pkg/server/chaosd/disk.go

```diff
@@ -35,87 +36,132 @@ type diskAttack struct{}
 var DiskAttack AttackType = diskAttack{}
 
 const DDWritePayloadCommand = "dd if=/dev/zero of=%s bs=%s count=%s oflag=dsync"
-const DDReadPayloadCommand = "dd if=%s of=/dev/null bs=%s count=%s iflag=dsync,direct,fullblock"
+const DDReadPayloadCommand = "dd if=%s of=/dev/null bs=%s count=%s iflag=dsync, fullblock, nocache"
```

Comment:
why need to update it?

Reply:
Direct iflag will forbid user from reading virtual device, so I use nocache to instead it which is  similar with direct.